### PR TITLE
Hint for Lemonade Line

### PR DESCRIPTION
**Pull request by brad182**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/silver/usaco-835.mdx [hunk 1]
```diff
@@ -4,6 +4,25 @@ source: USACO Silver 2018 Open
...
 ---
+<Spoiler title="Hint 1">
+
+Consider how you can process the cows to minimize the number of cows joining the line. Think about that possible situations where a cow would be willing to wait behind the others.
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+This problem is best solved with a greedy approach. Consider which data structures will be useful in being able to get the cow that is the most willing to wait in line.
```
Comment (SansPapyrus683) [inline solutions/silver/usaco-835.mdx:15]: people presumably come here from the greedy module so...

File: solutions/silver/usaco-835.mdx [hunk 2]
```diff
@@ -4,6 +4,25 @@ source: USACO Silver 2018 Open
...
+
+</Spoiler>
+
+<Spoiler title="Hint 3">
+
+Use a list and sort it using comparisons in each iteration of a loop. Keep going until there are no more cows with willingness values greater than or equal to the minimum number of cows that might join the line.
+
+</Spoiler>
```
Comment (SansPapyrus683) [inline solutions/silver/usaco-835.mdx:23]: ngl this is just the sol

File: solutions/silver/usaco-835.mdx [hunk 3]
```diff
@@ -4,6 +4,25 @@ source: USACO Silver 2018 Open
 title: Lemonade Line
 author: Mrinall Umasudhan, Ryan Chou
 ---
+<Spoiler title="Hint 1">
+
+Consider how you can process the cows to minimize the number of cows joining the line. Think about that possible situations where a cow would be willing to wait behind the others.
```
Comment (SansPapyrus683) [inline solutions/silver/usaco-835.mdx:9]: a bit vague?

File: solutions/silver/usaco-835.mdx (hunk 2 above)
Comment (ryanchou-dev) [reply solutions/silver/usaco-835.mdx:23]: ^^

**Review by ryanchou-dev — COMMENTED** [review]
this problem is kinda hard to write hints for bc it's not too complex...

maybe just have one hint that's like: "how do we make sure that the cows who are the most impatient have the least chance to get lemonade" or smtg

**Comment by brad182** [comment]
> this problem is kinda hard to write hints for bc it's not too complex...
> 
> maybe just have one hint that's like: "how do we make sure that the cows who are the most impatient have the least chance to get lemonade" or smtg

don't think that's true. maybe it's easy to you, but maybe not someone actively trying to improve on silver level problems

File: solutions/silver/usaco-835.mdx (hunk 3 above)
Comment (brad182) [reply solutions/silver/usaco-835.mdx:9]: how specific do you want the hints to be? i think i did that intentionally to get them to think and not give much away

**Comment by SansPapyrus683** [comment]
i think just delete hint 3 and we good?

**Comment by ryanchou-dev** [comment]
> > this problem is kinda hard to write hints for bc it's not too complex...
> > maybe just have one hint that's like: "how do we make sure that the cows who are the most impatient have the least chance to get lemonade" or smtg
> 
> don't think that's true. maybe it's easy to you, but maybe not someone actively trying to improve on silver level problems

that's fair! yeah just feels like some hints are restating the same thing. i agree with kevin, just delete hint3 and we're chilling B3

**Review by Aadit-Ambadkar — APPROVED** [review]
should be good after sans's revision

File: solutions/silver/usaco-835.mdx (hunk 1 above)
Comment (jessechoe10) [reply solutions/silver/usaco-835.mdx:15]: Yeah just remove the first part of the sentence

File: solutions/silver/usaco-835.mdx (hunk 2 above)
Comment (jessechoe10) [reply solutions/silver/usaco-835.mdx:23]: I agree

**Comment by SansPapyrus683** [comment]
deal

**Pull request merged**